src/ImageSharp/Formats/Jpeg/Components/Quantization.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SixLabors.ImageSharp.Formats.Jpeg.Components
+{
+    /// <summary>
+    /// Provides methods and properties related to jpeg quantization.
+    /// </summary>
+    internal static class Quantization
+    {
+        /// <summary>
+        /// Threshold at which given luminance quantization table should not be considered 'standard'.
+        /// </summary>
+        /// <remarks>
+        /// Jpeg does not define either 'quality' nor 'standard quantization table' properties
+        /// so this is purely a practical value derived from tests.
+        /// </remarks>
+        public const double StandardLuminanceTableVarianceThreshold = 10.0;
```

Comment:
This a WIP value, I will write tests which output variances with quality 1-100 for standard tables so we can use better value as threshold. Same for chroma threshold.